nautobot/extras/tests/test_views.py

```diff
@@ -1543,6 +1543,39 @@ def test_user_default_precedes_global_default(self):
         # Assert that Location List View got redirected to Saved View set as user default
         self.assertBodyContains(response, "<strong>User Location Default View</strong>", html=True)
 
+    @override_settings(EXEMPT_VIEW_PERMISSIONS=["*"])
+    def test_filtered_view_precedes_global_default(self):
+        view_name = "dcim:location_list"
+        SavedView.objects.create(
+            name="Global Location Default View",
+            owner=self.user,
+            view=view_name,
+            is_global_default=True,
+        )
+        response = self.client.get(reverse(view_name) + "?location_type=Campus", follow=True)
+        # Assert that the user is not redirected to the global default view
+        # But instead redirected to the filtered view
+        self.assertNotIn(
+            "<strong>Global Location Default View</strong>",
+            extract_page_body(response.content.decode(response.charset)),
+        )
```

Comment:
Wonder if instead/in addition we should make the saved view apply a different filter (e.g. `location_type=Floor`) and assert that any `Campus` location is shown in the response and no `Floor` location is shown in the response?